build.gradle

```diff
@@ -60,11 +66,11 @@ ext {
     commonsLang3Version = '3.8.1'
     hamcrestVersion = 1.3
     hibernateValidatorVersion = '6.0.14.Final'
-    junitJupiterVersion = '5.4.0'
-    junitVintageVersion = '5.4.0'
+    junitJupiterVersion = '5.3.1'
+    junitVintageVersion = '5.2.0'
     postgresqlVersion = '42.2.5'
```

Comment:
I reckon we could use the later version of Jupiter

Reply:
It failed on the newer version so I reverted back to this version known to work on definition-store